go.mod

```diff
@@ -3,30 +3,31 @@ module github.com/tendermint/tendermint
 go 1.13
 
 require (
-	github.com/ChainSafe/go-schnorrkel v0.0.0-20200115165343-aa45d48b5ed6
+	github.com/ChainSafe/go-schnorrkel v0.0.0-20200405005733-88cbf1b4c40d
 	github.com/Workiva/go-datastructures v1.0.52
 	github.com/btcsuite/btcd v0.20.1-beta
-	github.com/btcsuite/btcutil v1.0.1
+	github.com/btcsuite/btcutil v1.0.2
 	github.com/fortytw2/leaktest v1.3.0
 	github.com/go-kit/kit v0.10.0
 	github.com/go-logfmt/logfmt v0.5.0
 	github.com/gogo/protobuf v1.3.1
-	github.com/golang/protobuf v1.3.5
+	github.com/golang/protobuf v1.4.0
```

Comment:
there is a deprecation warning on this dep because it has been superseeded by google.golang.org/protobuf/proto, but we cannot upgrade due to gogo/protobuf not updating yet